modules/common/react/lib/theming/index.ts

```diff
@@ -1,4 +1,7 @@
+import CanvasProvider from './CanvasProvider';
+
 export * from './breakpoints';
 export * from './createCanvasTheme';
 export * from './types';
 export * from './theme';
+export {CanvasProvider};
```

Comment:
thoughts on moving `CanvasProvider` to common under theming? I might pair better with input provider.

Reply:
Let's keep it here. Honestly I wouldn't mind moving `InputProvider` into common in a separate PR as well.